packages/store/src/patterns/patternMatchers.js

```diff
@@ -307,29 +290,23 @@ const makePatternKit = () => {
         }
         const [specNames, specValues] = recordParts(specimen);
```

Comment:
Nit: "spec" as the prefix of a variable name is more suggestive to me of "specification" than "specimen".
```suggestion
        const [specimenNames, specimenValues] = recordParts(specimen);
```

Reply:
Done